openpype/plugins/publish/extract_review.py

```diff
@@ -1044,6 +1040,11 @@ def audio_args(self, instance, temp_data, duration_seconds):
                 path_to_subprocess_arg(audio["filename"])
             ))
 
+        # Add exclusive mapping for audio channels
+        audio_in_args.append("-map 0:v")
```

Comment:
I don't get why you need to add the video stream here.

Reply:
I had empty video channel without this when audio inputs are movie file.